homeassistant/components/shelly/utils.py

```diff
@@ -265,7 +265,7 @@ def get_rpc_channel_name(device: RpcDevice, key: str) -> str:
     if device.config.get("switch:0"):
         key = key.replace("input", "switch")
     device_name = get_rpc_device_name(device)
-    entity_name: str | None = device.config[key].get("name", device_name)
+    entity_name: str | None = device.config[key].get("name", device_name) if key in device.config else None
```

Comment:
```suggestion
    entity_name: str | None = None
    if key in device.config:
        entity_name = device.config[key].get("name", device_name)
```
The condition is too long so it fails black check and if a condition should be split to more than one line we don't use ternary condition.